src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/Location.kt

```diff
@@ -631,6 +631,16 @@ abstract class Location(
         dataChanged = true
       }
 
+      if (this is Cell) {
+        addHistory(
+          LocationAttribute.OPERATIONAL_CAPACITY,
+          getWorkingCapacityIgnoreParent().toString(),
+          "0",
```

Comment:
We've got "0" here but "None" in Cell.kt, is this correct?

Reply:
Yes only None when converted from non-cell